app/components/concept-admin/questions-page/question-form.hbs

```diff
@@ -54,11 +54,22 @@
                 Option #{{add optionIndex 1}}
               </div>
               <div class="flex items-center gap-2 opacity-0 group-hover/option-form:opacity-100 transition-opacity">
-                <TertiaryButton @size="extra-small" class="group/trash-button" @isDisabled={{eq optionIndex 0}} data-test-move-up-button>
+                <TertiaryButton
+                  @size="extra-small"
+                  class="group/trash-button"
```

Comment:
@libmartinito this naming seems like a mistake (from before) that we should correct. This isn't a "trash" button. 

```suggestion
                  class="group/move-up-button"
```